src/Identity/samples/IdentitySample.DefaultUI/Startup.cs

```diff
@@ -58,13 +58,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
 
             app.UseStaticFiles();
 
+            app.UseRouting();
+
             app.UseAuthentication();
+            app.UseAuthorization();
 
-            app.UseMvc(routes =>
+            app.UseEndpoints(endpoints =>
             {
-                routes.MapRoute(
-                    name: "default",
-                    template: "{controller=Home}/{action=Index}/{id?}");
+                endpoints.MapDefaultControllerRoute();
```

Comment:
Random observation:
It feels *weirder* to mix the setting up of routes on an "endpoints" argument compared to a "routes" argument.

Reply:
I don't think this can/should change. The metaphor from MVC is *routes* yet routes are not a concept in endpoint routing. It's intentionally names this way so you can easily transition code.

Let the past be the past.